_includes/seamless-view-events.md

```diff
@@ -0,0 +1,197 @@
+{% assign api_resource = include.api_resource %}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Could not find module `remark-inline-links`](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5637621)